# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/*/dvgrab/gcc34.patch
# Copyright (C) 1998 - 2004 ROCK Linux Project
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

I wonder why g++ ever accepted this code ...

  - Rene Rebe <rene@rocklinux.org>

--- dvgrab/ieee1394io.h.orig	2004-05-15 20:14:18.805136568 +0200
+++ dvgrab/ieee1394io.h	2004-05-15 20:16:24.957958424 +0200
@@ -33,7 +33,7 @@
 
 class IEEE1394Reader
 {
-private:
+protected:
 	/// the number of frames that had to be thrown away because
 	/// our inFrames queue did not contain available frames
 	static int droppedFrames;
